build.cake

```diff
@@ -8,19 +8,21 @@ var packageWhitelist = new[]
     MakeAbsolute(File("./src/Splat/Splat.csproj")),
     MakeAbsolute(File("./src/Splat.Autofac/Splat.Autofac.csproj")),
     MakeAbsolute(File("./src/Splat.DryIoc/Splat.DryIoc.csproj")),
-    MakeAbsolute(File("./src/Splat.Ninject/Splat.Ninject.csproj")),
-    MakeAbsolute(File("./src/Splat.SimpleInjector/Splat.SimpleInjector.csproj")),
     MakeAbsolute(File("./src/Splat.Log4Net/Splat.Log4Net.csproj")),
+    MakeAbsolute(File("./src/Splat.Microsoft.Extensions.DependencyInjection/Splat.Microsoft.Extensions.DependencyInjection.csproj")),
```

Comment:
Ordered this thing alphabetically

Reply:
You've duplicated up entries etc by the looks with NInject.

Reply:
Lemme fix that sorry.